features_and_tools/firely-server-ingest.rst

```diff
@@ -3,6 +3,10 @@
 Bulk Import via Firely Server Ingest
 ====================================
 
+.. note::
+
+  The features described on this page are available in **all** :ref:`Firely Server editions <vonk_overview>`.
```

Comment:
Should we mention the separate license token ``/unlimited``?

Reply:
@alexzautke : I see no reply nor a code change for this comment.

Reply:
Hmm, I see that the comment is in twice... Weird interplay of Rider and Github review probably.

Reply:
Yes, there were multiple comments twice